CHANGELOG.md

```diff
@@ -1,3 +1,151 @@
+# 108.0.0
+
+This major release includes several API additions to BottomSheet, Dialogs, and NavigationDrawer, along with a bug fix to Snackbar and improved Buttons documentation.
+
+## New deprecations
+
+MDCSemanticColorScheme's `init` was deprecated. Please use initWithDefaults: instead.
+
+## New features
+
+### Creating an `MDCAlertController` with attributed text
+
+```objc
+NSAttributedString *attributedStr =
+    [[NSAttributedString alloc] initWithString:@"attributed message" attributes:@{}];
+self.attributedAlert = [MDCAlertController alertControllerWithTitle:@"title"
+                                                  attributedMessage:attributedStr];
+```
+
+### Aligning subviews of an `MDCAlertControllerView`
```

Comment:
Not sure what "Aligning" mean. How about: "Dialogs adjustable insets" ?

Reply:
Updated to "Positioning subviews of an `MDCAlertControllerView`"

Reply:
Update: Positioning subviews of a Dialog